### PR TITLE
Fixes invalid Syntax in stub files

### PR DESCRIPTION
**Pull request by gtalarico**
TLDR: Importing rhino3dm causes mypy type checks to fail and cannot be ran on project

Not sure if this is automatically generated, but invalid syntax causes mypy to fail.

As per [PEP-484](https://www.python.org/dev/peps/pep-0484/#stub-files):
> [pyi] stub files are syntactically valid Python modules

I was able to fix the errors manually, which are shown in this PR.

The two issue types are: 
* Indent errors on functions and class with no body
* List types should be `List[int]` and `List[Curve]`. `int[]` and `Curve[]` are invalid intax

### Before Patching
Cannot run mypy type check on my project.
There might be a way of suppressing this error but I wasn't able to. 
I thought `--follow-imports=silent` would work but it did not.

```shell
.venv ❯ mypy myproject
.venv/lib/python3.7/site-packages/rhino3dm/__init__.pyi:949: error: invalid syntax
```

### After Patching
Type errors/warnings show as expected
```shell
.venv ❯ mypy by
contrib/rhino/conversion.py:14: error: Incompatible types in assignment (expression has type "Point3d", variable has type "Point")
contrib/rhino/conversion.py:15: error: Argument 1 to "Sphere" has incompatible type "Point"; expected "Point3d"
contrib/rhino/conversion.py:22: error: Incompatible return value type (got "LineCurve", expected "Line")
contrib/rhino/conversion.py:27: error: "Point3d" has no attribute "X"
```

### CODE REVIEW

**Comment by sbaer** [comment]
Thanks @gtalarico I wrote a console application that autogenerates this file at
https://github.com/mcneel/rhino3dm/tree/master/src/docgen
specifically at
https://github.com/mcneel/rhino3dm/blob/master/src/docgen/PythonClass.cs#L210

I'll try and update the generator to match your changes
